CAFAna/Experiment/ReactorExperiment.cxx

```diff
@@ -64,35 +70,54 @@ namespace ana
   }
 
   //----------------------------------------------------------------------
-  const ReactorExperiment* DayaBayConstraint2014()
-  {
-    // Daya Bay, Neutrino 2014:
-    // https://indico.fnal.gov/getFile.py/access?contribId=256&sessionId=15&resId=0&materialId=slides&confId=8022
-    return new ReactorExperiment(.084, .005);
-  }
-
-  //----------------------------------------------------------------------
-  const ReactorExperiment* WorldReactorConstraint2015()
+  const ReactorExperiment* ReactorConstraintNuFit2018(bool normal_ordering, bool sk_data)
   {
-    // Weighted average of
-    // Daya Bay: arXiv:1505.03456
-    // RENO: arXiv:1410.7987
-    // Double Chooz: 1406.7763
-    return new ReactorExperiment(.086, .005);
+    // NuFit November 2018
+    double this_val = 0., this_err = 0.;
+
+    if(sk_data){
+      if(normal_ordering)
+        this_val = 0.088, this_err = 0.003;
```

Comment:
Please don't use a comma like this. I guess it works, but it's super non-standard. Semicolon, and put the block in {}.